### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by SuperCHAOSXXX** [APGRoboCop/foxbot#29]
Hi, i'm running the latest version of FoxBot and i've noticed that after playing a certain amount of time(1-15 minutes), the game crashes and i get this error:

![Screenshot (29)](https://github.com/APGRoboCop/foxbot/assets/54687713/6f372f78-a1b6-41f4-abe4-e18ebd34ae02)

**Comment by ForeskinJohn** [comment]
0.77 was the last stable version of foxbot, for some reason any updates are broken. unless theres some sort of redistributable needed, which isnt mentioned anywhere here for whatever reason.

**Comment by Safety1st** [comment]
>  the game crashes and i get this error
Please next time provide more detailed bug report – with technical info about your environment. That greatly increases chances to get professional support at all.

For now I would firstly recommend to downgrade your TFC client to pre-anniversary build. I haven't tested the latest ones but I haven't noticed crashes at all either.

**Comment by Safety1st** [comment]
> the game crashes and i get this error

@SuperCHAOSXXX, plz next time provide more detailed bug report – with technical info about your environment. That greatly increases chances to get professional support at all.

For now I would firstly recommend to downgrade your TFC client to pre-anniversary build. I haven't tested the latest ones but I haven't noticed crashes at all either.

**Comment by ForeskinJohn** [comment]
im using pre anniversary, and i still get issues, will be testing 0.911 today

**Comment by Safety1st** [comment]
@ForeskinJohn, plz don't forget to share results with us 🙏

**Comment by APGRoboCop** [comment]
Yes well FoXBot isn't quite ready for HL1 25th Anniv mode yet, and the newer HLSDK for 25th Anniv may need some tweaks and bug fixes, as well as both the client and server builds.
